src/CrawlLogger.php

```diff
@@ -117,49 +118,71 @@ public function crawled(
         ResponseInterface $response,
         ?UriInterface $foundOnUrl = null
     ) {
-        $statusCode = $response->getStatusCode();
-
-        $reason = $response->getReasonPhrase();
-
-        $colorTag = $this->getColorTagForStatusCode($statusCode);
-
-        $timestamp = date('Y-m-d H:i:s');
-
-        $message = "{$statusCode} {$reason} - ".(string) $url;
+        // https://github.com/guzzle/guzzle/blob/master/docs/faq.rst#how-can-i-track-redirected-requests
+        if ($response->getHeader('X-Guzzle-Redirect-History')) {
+            // Retrieve both Redirect History headers
+            $fullRedirectReport = [];
+            // Retrieve both Redirect History headers
+            $redirectUriHistory = $response->getHeader('X-Guzzle-Redirect-History'); // retrieve Redirect URI history
+            $redirectCodeHistory = $response->getHeader('X-Guzzle-Redirect-Status-History'); // retrieve Redirect HTTP Status history
```

Comment:
Always but a comment above the line you're commenting on.

Add a blank line after each block the comment applies to.

This will make it much more readable.